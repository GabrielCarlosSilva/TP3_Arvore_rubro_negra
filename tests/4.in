1
6
Maria
58
Mauro
63
Kamila
29
Pedro
30
Alana
38
Luisa
19
2
0