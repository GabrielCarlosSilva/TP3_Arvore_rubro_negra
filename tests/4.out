Dados inOrder:
Nome: Luisa
Idade: 19
Nome: Kamila
Idade: 29
Nome: Pedro
Idade: 30
Nome: Alana
Idade: 38
Nome: Maria
Idade: 58
Nome: Mauro
Idade: 63